src/components/Portal/Portal.tsx

```diff
@@ -23,7 +24,8 @@ export interface IPortalProps {
   onUnmount?: (props: IPortalProps) => void
   open?: boolean
   trigger?: JSX.Element
-  triggerRef?: (node: HTMLElement) => void
+  trapFocus?: boolean
+  focusTrapZoneProps?: IFocusTrapZoneProps
```

Comment:
we should consider moving these two props to behavior. let's discuss

Reply:
yes, we can make it in future PR, once a PopupBehavior get merged

Reply:
I have created https://github.com/stardust-ui/react/issues/354 for this